components/list/list-item-role-mixin.js

```diff
@@ -31,6 +34,9 @@ export const ListItemRoleMixin = superclass => class extends superclass {
 		this._nested = (parent.slot === 'nested');
 		this._separators = separators || undefined;
 		this._extendSeparators = parent.hasAttribute('extend-separators');
+
+		if (!dragDropMultipleFlag) return;
+		this._dragMultiple = this.getRootList()?.hasAttribute('drag-multiple');
```

Comment:
This seems to be where we add these things that we want send through to the list item from the parent list. It wouldn't update if `drag-multiple` changed later, but perhaps that's not a huge concern?